app/javascript/widget/helpers/campaignHelper.js

```diff
@@ -0,0 +1,41 @@
+import { triggerCampaign } from 'widget/api/campaign';
+window.campaignTimers = [];
```

Comment:
Is there a way to avoid window object. I understand that we need a global object. Can we do it in a different way?

Reply:
Added class for campaign helper